src/middle/Stan_math_signatures.ml

```diff
@@ -252,6 +252,7 @@ let distributions =
   ; ([Lpdf; Log], "inv_wishart", [DMatrix; DReal; DMatrix], SoA)
   ; ([Lpdf; Log], "lkj_corr", [DMatrix; DReal], AoS)
   ; ([Lpdf; Log], "lkj_corr_cholesky", [DMatrix; DReal], AoS)
+  ; ([Lpdf; Log], "lkj_cov", [DMatrix; DVector; DVector; DReal], AoS)
```

Comment:
I agree it's unfortunate. I'd rather avoid the special case and have it consistently be true that you can only use `_lpdf` or `_lpmf`, especially since it is deprecated until "3.0", which could be a very long time from now.

Reply:
The idea is that they are "too breaking". We can't (or perhaps never tried to) provide an automated way of translating existing code to the new recommendation, so we don't want to remove them without a full major version bump.